codegen/codegen.go

```diff
@@ -947,40 +951,40 @@ func (cg *CodeGen) EmitFilesystemChainStmt(ctx context.Context, scope *parser.Sc
 	return so, nil
 }
 
-func (cg *CodeGen) EmitOptions(ctx context.Context, scope *parser.Scope, op string, stmts []*parser.Stmt, ac aliasCallback) (opts []interface{}, err error) {
+func (cg *CodeGen) EmitOptions(ctx context.Context, scope *parser.Scope, op string, stmts []*parser.Stmt, ac aliasCallback, chainStart interface{}) (opts []interface{}, err error) {
```

Comment:
`EmitOptions` should not have `chainStart`

Reply:
fixed